### DISCUSSION TITLE
Embed photo captions in EXIF metadata

### DISCUSSION THREAD

**Discussion by jan3zk** [gilesknap/gphotos-sync#437] [Ideas]
Might it be feasible to incorporate the preservation of photo captions from the description field located in the info screen of the Google Photos app? A similar implementation at https://github.com/jan3zk/gphotos_backup/blob/master/gphotos_backup.py stores Google Photo captions as EXIF metadata. Could a comparable technique be applied for this tool?

**Comment by gilesknap** [comment]
In principal this could apply to gphotos-sync. It is a departure from current behaviour which is to download the media from GooglePhotos unchanged. Since the Google Photos API makes changes to the files anyway (like stripping GPS and compressing)  this is not a massive problem.

I like the idea of preserving as much of the Photos Library metadata as is feasible.
